### PR TITLE
[SRP backport 21.2] [HDRP][URP] update manifest of SRP templates (#5799)

### PR DESCRIPTION
**Pull request by sebastienlagarde**
### Purpose of this PR
Backport https://github.com/Unity-Technologies/Graphics/pull/5799

---
### Testing status
To be done

---
### Comments to reviewers

For reviewver, this PR is using package 12.1.0 of SRP. We need to wait that the SRP2Core PR which include this bump land. Then we will merge this PR and publis/promote the template.

### CODE REVIEW

File: com.unity.template-hd/Packages/manifest.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
   "dependencies": {
     "com.unity.cinemachine": "2.8.0",
-    "com.unity.ide.rider": "3.0.7",
-    "com.unity.ide.visualstudio": "2.0.11",
-    "com.unity.ide.vscode": "1.2.3",
+    "com.unity.collab-proxy": "1.9.0",    
+    "com.unity.feature.development": "1.0.1",
     "com.unity.inputsystem": "1.1.0-pre.5",
     "com.unity.learn.iet-framework": "2.0.0",
     "com.unity.render-pipelines.high-definition": "12.0.0",
```
Comment (vicky-voin) [inline com.unity.template-hd/Packages/manifest.json:8]: The version in package.json seems to be different, it is likely going to be flagged by Yamato when it runs

File: com.unity.template-universal/Packages/manifest.json [hunk 2]
```diff
@@ -2,15 +2,10 @@
   "dependencies": {
     "com.unity.2d.sprite": "1.0.0",
     "com.unity.2d.tilemap": "1.0.0",
-    "com.unity.ads": "3.7.5",
+    "com.unity.ads": "3.7.1",
```
Comment (vicky-voin) [inline com.unity.template-universal/Packages/manifest.json:5]: Don't know if I missed it on the previous PR, do we need this package? (it is not one of the default packages we need to add)
If yes, it should be included in package.json as well and if not - would be great if we can remove it from the manifest.

**Review by vicky-voin — CHANGES_REQUESTED** [review]

File: com.unity.template-universal/Packages/manifest.json (hunk 2 above)
Comment (sebastienlagarde) [reply com.unity.template-universal/Packages/manifest.json:5]: looks like a bad merge from me, thanks for pointing it

File: com.unity.template-hd/Packages/manifest.json (hunk 1 above)
Comment (sebastienlagarde) [reply com.unity.template-hd/Packages/manifest.json:8]: good catch

**Review by vicky-voin — APPROVED** [review]

**Pull request merged**